### ISSUE TITLE
osc status doesn't show anything if I only have pods

### ISSUE THREAD

**Issue by thoraxe** [openshift/origin#1696]
```
[joe@ose3-master beta3]$ osc status
In project OpenShift 3 Demo (demo)

You have no services, deployment configs, or build configs. 'osc new-app' can be used to create applications from scratch from existing Docker images and templates.
[joe@ose3-master beta3]$ osc get pod
POD                 IP                  CONTAINER(S)        IMAGE(S)                    HOST                                    LABELS                 STATUS              CREATED
hello-openshift     10.1.0.7            hello-openshift     openshift/hello-openshift   ose3-master.example.com/192.168.133.2   name=hello-openshift   Running             56 seconds
[joe@ose3-master beta3]$ osc version
osc v0.4.3.0
kubernetes v0.13.1-dev-641-gf057a25
```

I'm not sure that that is desired.

**Comment by jwforres** [comment]
@smarterclayton we show standalone pods in the web console overview, osc status should probably behave similarly.  In the console we also explicitly exclude showing standalone pods that we know are build or deployment pods.

**Comment by smarterclayton** [comment]
How does someone find a deployment pod that fails?

----- Original Message -----

**Comment by jwforres** [comment]
Browse -> Pods doesn't hide anything

**Comment by smarterclayton** [comment]
Not going to be fixed in 3.0
